### PR TITLE
[TACHYON-1243] added a private constuctor for ticket

### PR DESCRIPTION
**Pull request by CruncherBigData**
Addresses issue: https://tachyon.atlassian.net/browse/TACHYON-1243

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by gpang** [comment]
ok to test

File: common/src/main/java/tachyon/Version.java [hunk 1]
```diff
@@ -23,6 +23,8 @@
 public final class Version {
   public static final String VERSION;
 
+  private Version(){}
```
Comment (jsimsa) [inline common/src/main/java/tachyon/Version.java]: please use space after `()`, so `private Version() {}`

**Comment by gpang** [comment]
I don't think your commits are linked to your Github account. Could you please set up your Github account by following the Github instructions https://help.github.com/articles/setting-your-email-in-git/ ? Thanks!

**Comment by jsimsa** [comment]
LGTM

**Comment by gpang** [comment]
Thanks for improvement!

**Comment by gpang** [comment]
LGTM

**Pull request merged**